scripts/deploy.sh

```diff
@@ -27,6 +27,12 @@ function run_ansible {
   openstack-ansible ${ANSIBLE_PARAMETERS} --forks ${FORKS} $@
 }
 
+apt-get install python-dev -y
+curl https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py -o /opt/get-pip.py
```

Comment:
ah nice, so pointing at the get-pip.py from pypa.io would probably work and prevent the double requirements we are seeing in the gate.